packages/gamut/src/GridForm/types.ts

```diff
@@ -11,7 +11,7 @@ export type BaseFormField<Value> = {
 };
 
 export type GridFormCheckboxField = BaseFormField<boolean> & {
-  description: string;
+  description: string | React.ReactNode;
```

Comment:
Fun fact, `string` is included in `React.ReactNode`!

```suggestion
  description: React.ReactNode;
```